### DISCUSSION TITLE
Shortcut to Toggle (Hide/Show) Widgets from Desktop

### DISCUSSION THREAD

**Discussion by tshrjn** [rxhanson/Rectangle#1278] [Ideas]
In macOS Sanoma, widgets for Desktop are introduced, which are super great. Sometimes, we want to de-clutter or simplify the desktop by hiding the icons & widgets on the desktop.

Currently there is a setting in Desktop & Stage Manager section of System preferences to toggle this.
Img Attached hereby,
<img width="478" alt="Screenshot 2023-10-14 at 3 07 16 AM" src="https://github.com/rxhanson/Rectangle/assets/8372098/519a974b-8551-4f58-babe-14822a1dcdf1">
<img width="480" alt="Screenshot 2023-10-14 at 3 07 12 AM" src="https://github.com/rxhanson/Rectangle/assets/8372098/9f18284b-0b6e-4f7f-9329-3937f3e76b6d">

**Comment by rxhanson** [comment]
Thanks for the feedback. This isn't really something I'd look to add into Rectangle since it is a little outside of the scope of the app and I try to keep it slim on purpose.
